src/mlpack/methods/randomized_svd/randomized_svd.hpp

```diff
@@ -149,4 +261,20 @@ class RandomizedSVD
 } // namespace svd
 } // namespace mlpack
 
+namespace mlpack {
+namespace cf {
+
+//! Factorizer traits of Regularized SVD.
+template<>
+class FactorizerTraits<mlpack::svd::RandomizedSVD>
+{
+ public:
+  //! Data provided to RandomizedSVD has to be cleaned.
+  static const bool UsesCoordinateList = false;
+  static const bool UsesRandomizedSVD = true;
```

Comment:
I think that things will change with the refactoring, but if we still have `FactorizerTraits` afterwards, I think maybe we should call this `UsesSVDTypeDecomposition` or something like that.  The real difference here is just that the factorizer returns some `U`, `s`, `V` matrices where s is a vector, and `U * diagmat(s) * V.t()` is equal to the original matrix.